### PR TITLE
Fixes #2 add table with x arguments and Fixes #1

### LINKED ISSUE #1: Change the namespace to use open-systems-pharmacology.org
@Yuri05 Is the namespace systems-biology.com used anywhere?

### LINKED ISSUE #2: Add TableWithXArgument

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: OSPSuite.SimModel.xsd [hunk 1]
```diff
@@ -131,6 +131,18 @@
 	</xs:complexType>
 	<xs:element name="TableFormulaWithOffset" substitutionGroup="osp:Formula" type="osp:TableFormulaWithOffsetType"/>
 
+	<xs:complexType name="TableFormulaWithXArgumentType">
```
Comment (msevestre) [inline OSPSuite.SimModel.xsd:134]: new type definition of the TAbleFormulaWithXArgument

File: OSPSuite.SimModel.xsd [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <!--<?xml version="1.0" encoding="UTF-8"?>-->
-<xs:schema targetNamespace="http://www.systems-biology.com" xmlns:osp="http://www.systems-biology.com" xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified">
+<xs:schema targetNamespace="http://www.open-systems-pharmacology.org" xmlns:osp="http://www.open-systems-pharmacology.org" xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified">
```
Comment (msevestre) [inline OSPSuite.SimModel.xsd:2]: about time we fix that too

**Comment by msevestre** [comment]
Going to merge as this is used as submodule anyways

**Comment by msevestre** [comment]
@Yuri05 With this change, I am getting some weird errors from SimModel.

```
SimModelNET.SimModelException : Unspecified error
```

Can you double check simmodel and update what needs to be updated? Thanks

**Comment by msevestre** [comment]
Forget that. Namespace were hardcode in OSPSuite.Core

**Comment by msevestre** [comment]
nope still some errors :)

File: OSPSuite.SimModel.xsd [hunk 3]
```diff
@@ -131,6 +131,18 @@
...
+			<xs:extension base="osp:FormulaType">
+				<xs:all>
+					<xs:element maxOccurs="1" minOccurs="1" name="Table" type="osp:ReferenceType"/>
+					<xs:element maxOccurs="1" minOccurs="1" name="XArgument" type="osp:ReferenceType"/>
+				</xs:all>
+			</xs:extension>
+		</xs:complexContent>
+	</xs:complexType>
+	<xs:element name="TableFormulaWithArgument" substitutionGroup="osp:Formula" type="osp:TableFormulaWithXArgumentType"/>
+
```
Comment (Yuri05) [inline OSPSuite.SimModel.xsd:145]: `<xs:element name="TableFormulaWithArgument"`
should also be TableFormulaWith**X**Argument

**Comment by msevestre** [comment]
This was already updated

**Comment by Yuri05** [comment]
ok

**Pull request merged**